Shared/Extensions/Parse/PFFileObject+Extensions.swift

```diff
@@ -12,12 +12,12 @@ import Combine
 
 extension PFFileObject: ImageDisplayable {
 
-    var userObjectId: String? {
+    var image: UIImage? {
```

Comment:
Taking userObjectId out of image displayable because it was too user specific. Instead it has a fileObject property.